package/setup.py

```diff
@@ -495,11 +485,8 @@ def dynamic_author_list():
           provides=['MDAnalysis'],
           license='GPL 2',
           packages=find_packages(),
-          package_dir={'MDAnalysis': 'MDAnalysis'},
-          ext_package='MDAnalysis',
```

Comment:
this was the final culprit that lead to setuptools trying to copy to a folder `MDAnalysis/MDAnalysis/...` which doesn't exist. @orbeckst changes broke our expected behavior. I would like to keep it like this now because it's more in line with other projects setup of cython.